modules/ipi-install-troubleshooting-installing-creates-no-worker-nodes.adoc

```diff
@@ -0,0 +1,40 @@
+// This module is included in the following assemblies: 
+//
+// installing/installing_bare_metal_ipi/ipi-install-troubleshooting.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="installing-creates-no-worker-nodes_{context}"]
+= Installing creates no worker nodes
+
+Unlike control plane nodes, the installation program does not provision worker nodes directly. Instead, the Machine API Operator scales nodes up and down on supported platforms. If worker nodes are not created after 15 to 20 minutes, depending on the speed of the cluster's internet connection, investigate the Machine API Operator.
```

Comment:
To me this currently suggests that The installation program doesn't provision worker node; unlike the control plane nodes which do provision worker nodes. Do we need the intro phrase? 
```suggestion
The installation program does not provision worker nodes directly. Instead, the Machine API Operator scales nodes up and down on supported platforms. If worker nodes are not created after 15 to 20 minutes, depending on the speed of the cluster's internet connection, investigate the Machine API Operator.
```